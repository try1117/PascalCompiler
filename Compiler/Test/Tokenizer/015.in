1e10 123 57 1.e3