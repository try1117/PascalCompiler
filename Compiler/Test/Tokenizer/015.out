0  | 0  | CONST_DOUBLE             | 1e10                     | 10000000000.000000000000000
0  | 5  | CONST_INTEGER            | 123                      | 123
0  | 9  | CONST_INTEGER            | 57                       | 57
0  | 12 | CONST_DOUBLE             | 1.e3                     | 1000.000000000000000
